### PR TITLE
Remove downshift dependency for switching controller

### PR DESCRIPTION
**Pull request by TingALin**
Replace downshift component with selected state
Rename the css wrapper links to downshift

### CODE REVIEW

**Comment by rossbulat** [comment]
Is this ready to be reviewed?

**Comment by rossbulat** [comment]
I can select accounts that do not have enough DOT.

screen

Can you disable these buttons when they are < ED. There should be a class already defined that changes the opacity when these are disabled. (the `outline` property should be `none` on these buttons - should prevent the blue border in the screenshot.

**Comment by TingALin** [comment]
> I can select accounts that do not have enough DOT.
> 
> screen
> 
> Can you disable these buttons when they are < ED. There should be a class already defined that changes the opacity when these are disabled. (the `outline` property should be `none` on these buttons - should prevent the blue border in the screenshot.

fixed

File: src/library/Form/AccountDropdown/Wrappers.ts [hunk 1]
```diff
@@ -96,6 +96,7 @@ export const StyledDropdown = styled.div<any>`
       &.inactive {
         opacity: 0.5;
         cursor: default;
+        border: none;
```
Comment (rossbulat) [inline src/library/Form/AccountDropdown/Wrappers.ts:99]: is this needed?

File: src/library/Form/AccountDropdown/index.tsx [hunk 2]
```diff
@@ -73,7 +73,9 @@ const DropdownItem = ({ item, onChange, selected }: any) => {
   const { unit } = useApi().network;
   return (
     <button
-      className={selected ? 'selected' : undefined}
+      className={`${selected ? 'selected' : undefined} ${
+        item.active ? '' : ' inactive'
```
Comment (rossbulat) [inline src/library/Form/AccountDropdown/index.tsx:77]:
```suggestion
!item.active ? `inactive` : ``
```

File: src/library/Form/AccountDropdown/Wrappers.ts (hunk 1 above)
Comment (TingALin) [reply src/library/Form/AccountDropdown/Wrappers.ts:99]: yes, as outline:none is not working, this one does

Comment (rossbulat) [reply src/library/Form/AccountDropdown/Wrappers.ts:99]: the button should just be disabled if not active, which will then make it not selectable.

**Comment by rossbulat** [comment]
All okay, good bye downshift.

**Pull request merged**